### ISSUE TITLE
updated map for val set

### ISSUE THREAD

**Issue by TechnoNeko** [johnwlambert/tbv#15]
Thank you for your work! I would like to know if the validation set provides the updated maps, meaning that the sensor images and the maps are consistent?

**Comment by TechnoNeko** [comment]
@johnwlambert @senselessdev1 @awayfromdog1  Do you have the updated map files? I really need them, and I would greatly appreciate it if you could share them

**Comment by johnwlambert** [comment]
Hi @TechnoNeko, thanks for your interest in TbV.

For many val split  logs, you can find the updated vector map by finding logs at a later chronological date that are clustered in the same area. However, not every Val log has an updated map. If you can’t find it, you could annotate the updated map yourself, or extract it from satellite imagery. 

https://github.com/johnwlambert/tbv/blob/main/tbv/scene_clusters.py

In the [AV2 readme](
https://argoverse.github.io/user-guide/datasets/map_change_detection.html#maps) we discuss how 

> A local vector map and a local ground height raster map is provided per log

You may also find the [TbV FAQ](https://github.com/johnwlambert/tbv?tab=readme-ov-file#faq) helpful which discusses this.

> A: Yes. Val split labels can be found [here](https://github.com/johnwlambert/tbv/blob/main/labeled_data/tbv_val_split_annotations.json). All train logs are positive logs that contain no changes. Most of the val and test logs contain at least some change (negatives), although some are positive "before" logs

> Q: Is there a file that indicates different log pairs and what has changed?

> A: We provide a clustering of logs by spatial location [here](https://github.com/johnwlambert/tbv/blob/main/tbv/scene_clusters.py). A few things to note:

> Each log within a cluster shares some significant visual overlap with other logs within its cluster.

> These are not necessarily before/after pairs. In some cases, all logs in a cluster may be "after" a change.

Each cluster has at least one log in the val or test set.
Logs of each cluster are provided in chronological order.
